.ansible-lint

```diff
@@ -0,0 +1,4 @@
+---
+skip_list:
+- '204'
+- '602'
```

Comment:
Yes, in my opinion we should skip both rules.
But I'd like to see ansible-lint config file inside molecule directory, something like this in molecule/default/molecule.yml would work:
  lint:
    name: ansible-lint
    options:
      c: molecule/default/ansible-lint.yml

Reply:
I think we should keep rule 204 (line length)

Reply:
In that case the line tasks/main.yml:11 needs to be fixed.

Reply:
which is easy, you can either make it a list (it's all and'ed) or split the line.  https://docs.ansible.com/ansible/latest/user_guide/playbooks_conditionals.html#the-when-statement

Reply:
`.ansible-lint` used in top level directory positively affects role score in ansible galaxy so I don't think it should live in molecule subdir.

Reply:
> which is easy, you can either make it a list (it's all and'ed) or split the line. https://docs.ansible.com/ansible/latest/user_guide/playbooks_conditionals.html#the-when-statement

... which, by the way, is what #36 does (thanks, @paulfantom, I knew I saw it somewhere :-) )

Reply:
> `.ansible-lint` used in top level directory positively affects role score in ansible galaxy so I don't think it should live in molecule subdir.

Good point. Another point is, if we are going to share molecule config among the roles, having .ansible-lint outside will allow for differences among the roles (ideally they would all follow the same standards, in practice different roles may need different exceptions though)

Reply:
My practice from @cloudalchemy shows that with time `.ansible-lint` is going to be different across all repos. Ideally there should be no such file and role should follow all guidelines, but in some cases this is problematic (usually this is a problem with long lines).

As for sharing `molecule` directory. Are you planning on doing multi-scenario tests and doing any pytest/testinfra validations? Because those also tend to be repo-specific.

Reply:
Also with #36 you won't need rule 204 to be excluded and rule 602 is "problematic" (more here: https://github.com/ansible/ansible-lint/issues/457)

Reply:
> As for sharing `molecule` directory. Are you planning on doing multi-scenario tests and doing any pytest/testinfra validations? Because those also tend to be repo-specific.

No. We decided that this is out of scope for our Molecule setup.